### PR TITLE
[WIP] Add support for testing Hypershift on a disconnected env

### PR DESCRIPTION
**Pull request by assafad**
Add support for testing Hypershift on a disconnected env on AWS.

### CODE REVIEW

File: ci-operator/jobs/hypershift/hypershift-tests/hypershift-hypershift-tests-master-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- agent: kubernetes
+  cluster: build05
+  cron: 0 12 * * 5
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+    - base_ref: master
+      org: quay
+      repo: quay-tests
```
Comment (assafad) [inline ci-operator/jobs/hypershift/hypershift-tests/hypershift-hypershift-tests-master-periodics.yaml:11]: @tiraboschi Should be changed to:
```
org: openshift
repo: hypershift
```
?

Comment (tiraboschi) [reply ci-operator/jobs/hypershift/hypershift-tests/hypershift-hypershift-tests-master-periodics.yaml:11]: yes, +1

File: ci-operator/jobs/hypershift/hypershift-tests/hypershift-hypershift-tests-master-periodics.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  spec:
+    containers:
+      - args:
+          - --gcs-upload-secret=/secrets/gcs/service-account.json
+          - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+          - --lease-server-credentials-file=/etc/boskos/credentials
+          - --oauth-token-path=/usr/local/github-credentials/oauth
+          - --report-credentials-file=/etc/report/credentials
+          - --secret-dir=/secrets/ci-pull-credentials
+          - --secret-dir=/usr/local/quay-omr-tests-omr-ocp412-disconnected-cluster-profile
```
Comment (tiraboschi) [inline ci-operator/jobs/hypershift/hypershift-tests/hypershift-hypershift-tests-master-periodics.yaml:40]: probably we will have also to handle this

**Pull request closed without merging**